src/main/java/com/cloudbees/jenkins/support/impl/JenkinsLogs.java

```diff
@@ -411,8 +423,20 @@ private void addMasterJulLogRecords(Container result) {
             }
         });
 
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins == null) {
+            LOGGER.log(Level.WARNING, "Cannot add master logs to the bundle. Jenkins instance is not ready");
+            return;
+        }
+        
+        final File[] logFiles = jenkins.getRootDir().listFiles(ROTATED_LOGFILE_FILTER);
+        if (logFiles == null) {
+            LOGGER.log(Level.WARNING, "Cannot add master logs to the bundle. Cannot access log files");
+            return;
+        }
+        
         // log records written to the disk
-        for (File file : SupportPlugin.getRootDirectory().listFiles(new LogFilenameFilter())){
+        for (File file : logFiles){
```

Comment:
:bug: this appears to be changing the implementation to something different and incorrect for no apparent reason. I think you want

```java
for (File file : ArrayUtil.fromList(SupportPlugin.getRootDirectory().listFiles(new LogFilenameFilter())) {
```